library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+        ** Extreme-delta-brush <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern}[(Source: Beniczky ea 2017, Table 5.)]</nowiki>
+
+
+'''Critically-ill-patients-patterns''' <nowiki> [Rhythmic or periodic patterns in critically ill patients (RPPs) are scored according to the 2012 version of the American Clinical Neurophysiology Society Standardized Critical Care EEG Terminology (Source: Hirsch ea 2013; Beniczky ea 2017, Section 9).]</nowiki>
+    * Critically-ill-patients-periodic-discharges <nowiki> {suggestedTag=RPP-morphology, suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Finding-frequency, suggestedTag=RPP-time-related-features} [Periodic discharges (PDs): Periodic - repetition of a waveform with relatively uniform morphology and duration with a quantifiable inter-discharge interval between consecutive waveforms and recurrence of the waveform at nearly regular intervals. (Source: Hirsch ea 2013; Suggested tags from Beniczky ea 2017, Table 8.)]</nowiki>
+    * Rhythmic-delta-activity <nowiki> {suggestedTag=RPP-with-superimposed-activity, suggestedTag=RPP-absolute-amplitude, suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Finding-frequency, suggestedTag=RPP-time-related-features} [Rhythmic Delta Activity (RDA): Rhythmic - repetition of a waveform with relatively uniform morphology and duration, and without an interval between consecutive waveforms. RDA - rhythmic activity less than or equal to 4 Hz. The duration of one cycle (i.e., the period) of the rhythmic pattern should vary by less than 50 percent from the duration of the subsequent cycle for the majority (greater than 50 percent) of cycle pairs to qualify as rhythmic. (Source: Hirsch ea 2013; Suggested tags from Beniczky ea 2017, Table 8.)]</nowiki>
+    * Spike-or-sharp-and-wave <nowiki> {suggestedTag=RPP-sharpness, suggestedTag=Number-of-RPP-phases, suggestedTag=RPP-triphasic-morphology, suggestedTag=RPP-absolute-amplitude, suggestedTag=RPP-relative-amplitude, suggestedTag=RPP-polarity, suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Multifocal-finding, suggestedTag=Finding-frequency, suggestedTag=RPP-time-related-features} [Spike-and-wave or Sharp-and-wave (SW) - polyspike, spike or sharp wave consistently followed by a slow wave in a regularly repeating and alternating pattern (spike-wave-spike-wave-spike-wave), with a consistent relationship between the spike (or polyspike or sharp wave) component and the slow wave; and with no interval between one spike-wave complex and the next (if there is an interval, this would qualify as PDs, where each discharge is a spike-and- wave). (Source: Hirsch ea 2013; Suggested tags from Beniczky ea 2017, Table 8.)]</nowiki>
```

Comment:
This is hard to parse.  Maybe:
Spike-or-sharp-and-wave  ->  Wave-with-spike-or-sharp

Reply:
Can'c change as we need to maintain the epilepsy terminology/definition, which is Spike-and-wave or Sharp-and-wave (SW).